src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,9 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        for (int i = 1; i <= 3; i++) {
+            System.out.println(lottery.getRandomBall().toString());
```

Comment:
toString() method is automatically called when we pass objects to System.out.println
```suggestion
            System.out.println(lottery.getRandomBall());
```